esp-mbedtls/src/lib.rs

```diff
@@ -367,6 +369,8 @@ pub mod asynch {
             mode: Mode,
             min_version: TlsVersion,
             certs: Option<&str>,
+            client_cert: Option<&str>,
+            client_key: Option<&str>,
```

Comment:
Should the handling of certificates be moved to a struct, or we had these 2 new arguments to the function signature?